kyuubi-hive-jdbc-shaded/pom.xml

```diff
@@ -148,6 +148,11 @@
             <artifactId>commons-lang</artifactId>
         </dependency>
 
+        <dependency>
+            <groupId>org.apache.commons</groupId>
+            <artifactId>commons-lang3</artifactId>
```

Comment:
You should relocate it and update the NOTICE

Reply:
Thanks @pan3793 .
NOTICE file already contains Apache Commons Lang.
![image](https://user-images.githubusercontent.com/70557521/163560367-05e87367-b18d-41ec-adcc-5fd91d2fffd9.png)
I have relocated  commons-lang3 and add its lisence.